templates/zerver/help/deactivate-or-reactivate-a-user.md

```diff
@@ -1,29 +1,47 @@
 # Deactivate or reactivate a user
 
+## Deactivating a user
+
 {!admin-only.md!}
 
-## Deactivate (ban) a user
+When you deactivate a user:
+
+* The user will be immediately logged out of all Zulip sessions, including
+  desktop, web and mobile apps.
+
+* The user's credentials for logging in will no longer work, including password
+  login and [any other login options](/help/configure-authentication-methods)
+  enabled in your organization .
+
+* The user's [bots](/help/bots-and-integrations) will be deactivated.
 
-To properly remove a user’s access to a Zulip organization, it does not
-suffice to change their password or deactivate their account in an external
-email system, since the user’s API key and bot API keys will still be
-active. Instead, you need to deactivate the user’s account using the Zulip
-administrative interface.
+* [Email invitations and invite links](/help/invite-new-users) created by the
+  user will be disabled.
 
-Note that organization administrators cannot deactivate organization owners.
+* Even if your organization [allows users to join without an
+  invitation](/help/restrict-account-creation#set-whether-invitations-are-required-to-join),
+  this user will not be able to rejoin with the same email account.
+
+!!! warn ""
+    You must go through the deactivation process below to fully remove a user's
+    access to your Zulip organization. Changing a user's password or removing
+    their single sign-on account will not log them out of their open Zulip
+    sessions, or disable their API keys.
+
+### Deactivate a user
 
 {start_tabs}
 
 {tab|via-user-profile}
 
-1. Hover over a user's name in the right sidebar.
+{!manage-this-user.md!}
 
-1. Click on the ellipsis (<i class="zulip-icon zulip-icon-ellipsis-v-solid"></i>)
-   to the right of their name.
+1. Click the **Deactivate user** button at the bottom.
 
-1. Click **Manage this user**.
+1. Configure **Notify this user by email?** as desired.
 
-1. Click the **Deactivate user** button at the bottom.
+1. If notifying the user, you can choose to enter a custom comment that will be
+   included in the notification email.
```

Comment:
I think we should consider combining this step with the previous step. Maybe we should start with "Select" or "Check the box" instead of "Configure", and say something like:

1. _(Optionally)_  Check the box **Notify this user by email?**, and you can also enter a custom comment to include it in the notification email.

```suggestion
1. _(Optionally)_  Check the box **Notify this user by email?**, and
   you can also enter a custom comment to include it in the notification email.
```

Reply:
Trying:

> 1. *(optional)* Select **Notify this user by email?** if desired, and enter a
   custom comment to include in the notification email.